canonical-websites/snapcraft-flask
# Snapcraft header

Vanilla styling for header

Fixes #15

## QA
- Update local branch
- Run `FLASK_APP=app.py FLASK_DEBUG=true flask run`
- Go to some snap page: http://localhost:5000/canonical-livepatch/
- Header should be vanilla navigation light pattern with snapcraft.io logo
 
<img width="1022" alt="screen shot 2017-09-05 at 09 14 38" src="https://user-images.githubusercontent.com/83575/30049423-e6eacd7a-921a-11e7-8caa-544792bde683.png">
